tests/test_checks.py

```diff
@@ -280,43 +280,43 @@ def test_monotonic_items():
         df), df + 1)
 
 
-def test_within_set():
+def test_has_vals_within_set():
     df = pd.DataFrame({'A': [1, 2, 3], 'B': ['a', 'b', 'c']})
     items = {'A': [1, 2, 3], 'B': ['a', 'b', 'c']}
-    tm.assert_frame_equal(df, ck.within_set(df, items))
+    tm.assert_frame_equal(df, ck.has_vals_within_set(df, items))
     tm.assert_frame_equal(df, dc.WithinSet(items=items)(_noop)(df))
```

Comment:
The decorator names will all change as well. Decorators are automatically generated with names with match the check version converted to title case. For example,`has_vals_within_set`'s decorator will be `HasValsWithinSet`

Reply:
We should actually probably have tests for both the deprecated names and the new ones for now, too.